### ISSUE TITLE
Side-by-side view doesn't allow copying

### ISSUE THREAD

**Issue by sirnacnud** [ForkIssues/Tracker#101]
When in the side-by-side view for showing the changes to a file, I can select text with the mouse, but the keyboard shortcut to copy the text doesn't work. Edit->Copy is also disabled, as well as the right click menu when right clicking on the select text.

Sometimes you may want to pull in parts of a commit and it would be nice to be able to copy from the side-by-side view.

**Comment by DanPristupov** [comment]
Yes, I'm aware about this problem. This happens because the real focus stays in the Fork window. It's a feature which allows you to use arrow keys to move the selection in the file list and switch from file to file. Similar to how it works in Finder.

However, there's a related issue. Since the focus is not in the pop-up, the `Cmd+C` shortcut can't be handled. Finder also has this problem, and Apple solved it by disabling selection in the preview window ;).

I need to figure out some hack/workaround which will allow to handle `Cmd+C` hotkey, while other window is selected.

**Comment by DanPristupov** [comment]
Fixed this bug is Fork 1.0.57 which is available now as prerelease. More information here: #110
